src/controllers/dashboard/library.js

```diff
@@ -218,29 +218,29 @@ import 'emby-itemrefreshindicator';
             name: '',
             value: ''
         }, {
-            name: globalize.translate('FolderTypeMovies'),
+            name: globalize.translate('Movies'),
             value: 'movies',
             message: getLink('MovieLibraryHelp', 'https://docs.jellyfin.org/general/server/media/movies.html')
         }, {
-            name: globalize.translate('FolderTypeMusic'),
+            name: globalize.translate('TabMusic'),
```

Comment:
I didn't feel like replacing the key everywhere it shows up but let me know if this is a blocker.